website/docs/r/cosmosdb_account.html.markdown

```diff
@@ -148,6 +150,16 @@ The following arguments are supported:
 * `id` - (Required) The ID of the virtual network subnet.
 * `ignore_missing_vnet_service_endpoint` - (Optional) If set to true, the specified subnet will be added as a virtual network rule even if its CosmosDB service endpoint is not active. Defaults to `false`.
 
+---
+
+A `backup_policy` block supports the following:
+
+* `type` - (Optional) The type of the `backup_policy`. Possible values are `Continuous` and `Periodic`. Defaults to `Periodic`.
```

Comment:
Shall we make this a `Required` property so that we can invalidate the usage of an empty `backup_policy` block?